saagieapi/jobs/gql_queries.py

```diff
@@ -143,6 +204,7 @@
     }){
         id
         name
+        alias
```

Comment:
The doc for the section `Editing a job` should be updated with the new GQL query (add field `alias` in the example